src/components/pages/service/product-design.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ContentBlockContainer } from '../../layout/content-block-container';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+import { Intro } from './support';
+
+export const ProductDesign = () => {
+  const { t } = useTranslation();
+
+  return (
+    <ContentBlockContainer>
+      <Intro
+        illustration="eclipse_039"
+        headline={t('services.product_design.title')}
+      >
+        <p>{t('services.product_design.text')}</p>
+      </Intro>
+    </ContentBlockContainer>
+  );
+};
```

Comment:
Even though the component is very small, I like that you put it in an extra file. 
This way everything is well structured 🙌